packages/grid/stories/grid-cellCustomization.stories.tsx

```diff
@@ -55,6 +57,30 @@ const PercentageCellValue = (props: GridCellValueProps<CurrencyPairRow>) => {
   );
 };
 
+const ButtonsCellValue = () => {
```

Comment:
Would be good to show how to get value from the row data as well.

Reply:
Check out the last commit, that good?

Reply:
👍